src/components/GameView/GameOverlays.vue

```diff
@@ -108,7 +108,6 @@ export default {
       'opponent',
       'hasGlassesEight',
       'player',
-      'resolvingSeven',
```

Comment:
Unused getter

Reply:
Technically we should do this in a separate PR to keep the changesets siloed but I’ll leave it up to @itsalaidbacklife.

Reply:
I'd rather just do it now --save bookkeeping. It's more effort to document that we ought to do this than to just do it